runtime/tests/interpreter/interpreter_test.go

```diff
@@ -7567,6 +7568,205 @@ func TestInterpretVariableDeclarationSecondValue(t *testing.T) {
 	)
 }
 
+func TestInterpretResourceMovingAndBorrowing(t *testing.T) {
```

Comment:
@SupunS These tests are failing. Did I by chance accidentally re-introduce them incorrectly?

Reply:
I think yes, this test was moved to checker tests in https://github.com/onflow/cadence/pull/2086, since now it fails at the checker.
We can simply remove this one here.

Reply:
Removed in [25a70a4](https://github.com/onflow/cadence/pull/2294/commits/25a70a461b405e0457d22af6bb84408fa2ab29d6)